packages/loader/container-loader/src/containerStorageAdapter.ts

```diff
@@ -90,6 +90,10 @@ export class ContainerStorageAdapter
 	) {
 		this._storageService = new BlobOnlyStorage(detachedBlobStorage, logger);
 		this._summarizeProtocolTree = shouldSummarizeProtocolTree;
+		this.logger.sendTelemetryEvent({
+			eventName: "isSummarizeProtocolTreeEnabled",
+			value: this._summarizeProtocolTree,
```

Comment:
Do you know if `Data_value` in our 1P partner's telemetry DB has data type boolean?  If not, this won't work.

Foolproof way would be to do `details: { value: this._summarizeProtocolTree }`.  And think about how it will appear when it's unset, and if you want to do `?? "UNSET"` or something explicit (not necessary IMO but worth considering)

Reply:
Maybe you don't even need to log here if it's undefined?  Then you'll just get at most 1 log per session, at whatever point the value is known.